docs/get-involved/community.md

```diff
@@ -5,42 +5,53 @@ title: Join the Community
-can find a list of our available rooms. The address for our homeserver is **matrix.radicle.xyz**.
+Most conversations in the community are happening here. The core team is very active and typically responds to questions
+in real time.
 
-→ [**#general**][gn] for introductions and general discussions about Radicle development
+[discord.com](https://discord.gg/dZK4TxaU2v)
 
-→ [**#support**][sp] for answering questions and other support-related discussions
+## [Discourse](https://radicle.community/)
 
-→ [**#gov**][gv] for discussing Radicle governance proposals and process
+The hub for all longer-form discussions about the project. The team uses it regularly to share project-wide
+announcements, releases, feature updates, and more.
 
-→ [**#token**][tn] for all conversation regarding the Radicle token
+[radicle.community](https://radicle.community/)
 
+## [Twitter](https://twitter.com/radicle)
 
-### Join our [community forum][cf]
-_For longer-form discussions on feature requests, development
-decisions, and other general topics_
+We use our Twitter to share announcements & project-wide updates, as well as amplify Radicle-aligned messaging from
+partnerships and the greater ecosystem. We don't offer support here.
 
-### Subscribe to our [telegram channel][tc]
-_For announcements from the Radicle project_
+[@radicle](https://twitter.com/radicle)
 
-### Follow us on twitter - [@radicle][tl]
-_For announcements and release updates_
+## [GitHub](https://github.com/radicle-dev)
 
-### Subscribe to our [email list][el] 
-_For announcements and release updates_
+GitHub
 
-[mc]: https://matrix.radicle.community/
-[sp]: https://matrix.to/#/#support:radicle.community
-[gn]: https://matrix.to/#/#general:radicle.community
-[gv]: https://matrix.to/#/#gov:radicle.community
-[tn]: https://matrix.to/#/#token:radicle.community
-[ch]: https://matrix.to/#/+home:radicle.community
-[cf]: https://radicle.community/
-[tl]: https://twitter.com/radicle
-[el]: http://eepurl.com/hhHxMX
-[tc]: https://t.me/radicleworld
+All Radicle development is open-source and public by default. The radicle-dev GitHub is consistently active and is the
+best place to see what the team is working on.
+
+[github.com/radicle-dev](https://github.com/radicle-dev)
+
+## [Matrix](https://matrix.radicle.community/)
+
+Some of the core team uses Matrix, and the Discord channels are mirrored here. Information on Matrix is almost the same
+as Discord.
+
+[matrix.radicle.community](https://matrix.radicle.community/)
+
+## [Telegram](https://t.me/radicleworld)
+
+Larger announcements are posted in our official Telegram channel. There are also other channels sprouting up which are
+quite active.
+
+[t.me/radicleworld](https://t.me/radicleworld)
+
+## [Newsletter](http://eepurl.com/hhHxMX)
+
+If you'd like to stay updated about product releases, project updates, and other announcements, hand us your e-mail and
+we'll keep you in the loop.
+
+[Stay up to date](http://eepurl.com/hhHxMX)
```

Comment:
Might be worth also mentioning that most of the radicle-link protocol development discussions happen on this mailing list: https://lists.sr.ht/~radicle-link/dev.